app/cdap/components/Connections/Create/ConnectionConfiguration/ConnectionConfigForm.tsx

```diff
@@ -68,7 +68,7 @@ export function ConnectionConfigForm({
     const fieldErrors = {};
     for (const [fieldName, value] of Object.entries(connectorProperties)) {
       if (
-        !!value.required &&
+        !!(value as any).required &&
```

Comment:
I think if you use `value.hasOwnProperty(required) && value.required`, you won't get the TS compiler error. Haven't tested it though

Reply:
<img width="851" alt="image" src="https://user-images.githubusercontent.com/20428112/165156475-2cab90b9-6dfb-4085-ac54-c3acef20ead4.png">
hmmm, it still shows up